src/JSInterop/Microsoft.JSInterop.JS/src/tsconfig.json

```diff
@@ -8,7 +8,8 @@
         "lib": ["es2015", "dom", "es2015.promise"],
         "strict": true,
         "declaration": true,
-        "outDir": "dist"
+        "outDir": "dist",
+        "module": "ESNext",
```

Comment:
This is needed because otherwise the TypeScript compiler interferes with `import` and tries to replace it with some old-school TypeScript-specific emulation of dynamic import. We don't want that - we just want the regular native import feature to be left intact.